resources/js/components/global/DashboardNotification.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div class="dashboard-notification padding-notification">
```

Comment:
No need to have two classes here. Get rid of ".padding-notification" and merge those styles in with ".dashboard-notification"